composer.json

```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": ">=5.4",
-        "cakephp/cakephp": "~3.0"
+        "cakephp/cakephp": "dev-master"
```

Comment:
Hmm without relying on dev-master the tests fail because the stub console output is not yet in a packaged release.